addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -1,6 +1,20 @@
 $o-we-overlay-zindex: ($zindex-fixed + $zindex-modal-backdrop) / 2 !default;
 $o-we-zindex: $o-we-overlay-zindex + 1 !default;
 
+// = Floating Toolbar Variables
+// ----------------------------------------------------------------------------
```

Comment:
Maybe remove those 79 ( :shrug: ) characters long fancy lines and add normal comments? `// Floating toolbar variables`

Same everywhere